VTUL/dlp-access
# Add language and availableAttributes files to codebase and import instead of fetching from s3

**Add language and availableAttributes files to codebase and import instead of fetching from s3.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2501) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Add language and availableAttributes files to codebase and import instead of fetching from s3.

# What's the changes? (:star:)
* Combine the two `lang_by_*.json` files into one and add to codebase
* Add `availableAttributes.json` to codebase
* Refactor `fetchTools.js` to import files from codebase instead of fetching from s3.
* Remove lang attribute from Site model (backend)

# How should this be tested?
* Check that search page, Archive show, and Collection pages are still working properly.
* Check that available attributes are loading correctly in the `Displayed Attributes` tab of the Site Admin page

# Additional Notes:
* branch: `LIBTD-2501`
* Backend model change required

# Interested parties
@yinlinchen 

(:star:) Required fields
